### PR TITLE
LPD-25440 Avoid cadmin classes in cookies banner configuration

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi,

According to [PTR-4886](https://liferay.atlassian.net/browse/PTR-4886) cadmin classes should not being applied to Cookies Banner Configuration since they make more difficult to be able to customize it.

Thanks.

Regards.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

File: modules/test/playwright/tests/cookies-banner-web/cookiesBannerCadmin.spec.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		});
+
+		if (await saveButton.isVisible()) {
+			await saveButton.click();
+		}
+		else if (await updateButton.isVisible()) {
+			await updateButton.click();
+		}
+
+		await waitForSuccessAlert(page);
```
Comment (dsanz) [inline modules/test/playwright/tests/cookies-banner-web/cookiesBannerCadmin.spec.ts:53]: I was about to suggest to use a .config file for this, but then I realized this snippet is similar to the one in another test file. I think we can live with this duplication for some time

**Comment by dsanz** [comment]
LGTM ;)

**Comment by antonio-ortega** [comment]
ci:forward

**Pull request closed without merging**